src/sheldon_dictionary.erl

```diff
@@ -168,3 +168,30 @@ typo_model(Word) ->
                 end,
                 [Word],
                 [lists:split(I, Word) || I <- lists:seq(0, length(Word))]).
+
+-spec generate_dictionary(term()) -> binary().
+generate_dictionary(Source) ->
+    DefaultDictionary = application:get_env(sheldon, default_dictionary, undefined),
+    AdditionalDictionaries = application:get_env(sheldon, additional_dictionaries, []),
+    case {DefaultDictionary, AdditionalDictionaries} of
+        {undefined, []} ->
+            {ok, SourceBin} = file:read_file(Source),
+            SourceBin;
+        {undefined, [_ | _] = AdditionalDictionaries} ->
+            concat_dictionaries(Source, AdditionalDictionaries);
+        {[_ | _] = DefaultDictionary, []} ->
```

Comment:
```suggestion
        {DefaultDictionary, []} ->
```
Filenames can be binaries, among other things… It's not necessary to use the robot butt here nor in the next clause.